src/calculator.js

```diff
@@ -1,47 +1,32 @@
-exports._check = () => {
+exports._check = (...args) => {
   // DRY up the codebase with this function
   // First, move the duplicate error checking code here
   // Then, invoke this function inside each of the others
   // HINT: you can invoke this function with exports._check()
+  args.forEach(arg => {
```

Comment:
Expected parentheses around arrow function argument having a body with curly braces.

###### See http://eslint.org/docs/rules/arrow-parens for details about this rule